rust_src/src/buffers.rs

```diff
@@ -924,6 +924,18 @@ impl From<LispBufferOrCurrent> for LispBufferRef {
     }
 }
 
+/// Return false.
+/// If the optional arg BUFFER is provided and not nil, enable undoes in that
+/// buffer, otherwise run on the current buffer.
+#[lisp_fn(min = "0")]
```

Comment:
We need to add `intspec = ""` here.

Reply:
I missed that. Thanks.

@odontomachus there is a tool in rust_src/admin called `rustize-emacs` which auto mates the creation of the Rust lisp_fn macro and the Rust function. It handles the interactive check for you as well as the min= if needed.